libraries/core/src/main/kotlin/com/bumble/appyx/core/node/ParentNode.kt

```diff
@@ -261,11 +261,13 @@ abstract class ParentNode<NavTarget : Any>(
 
     private class PermanentChildRender(private val node: Node) : ChildRenderer {
 
+        @Suppress("ComposableNaming")
         @Composable
         override fun invoke(modifier: Modifier) {
```

Comment:
Can you please add operator here?
```suggestion
        override operator fun invoke(modifier: Modifier) {
```

Reply:
done